app/api/schema/access_codes.py

```diff
@@ -27,15 +27,19 @@ class Meta:
 
     @validates_schema(pass_original=True)
     def validate_date(self, data, original_data):
+        ends_at = data.get('valid_till', None)
```

Comment:
Why initializing here

Reply:
changing same for access code too, like discount code